.travis.yml

```diff
@@ -13,6 +13,18 @@ env:
 # Build recommended versions based on: http://kafka.apache.org/downloads
 matrix:
   include:
+  - scala: 2.10
```

Comment:
we should update the readme instead of adding these versions back. reducing the number of supported versions helps us manage external limits like travis credits and docker pulls.

Reply:
What are your thoughts behind a 1-time push of those versions to the Docker registry and I'll submit a PR removing them and updating the readme?

Reply:
no one asked me, but I think that'd be great @drewpc

Reply:
I've removed the old versions again and updated the readme.